AppCenterDataStorage/AppCenterDataStorage/MSDocumentWrapper.h

```diff
@@ -0,0 +1,29 @@
+#import "MSSerializableDocument.h"
+
+@class MSDataSourceError;
+
+@interface MSDocumentWrapper<T : id <MSSerializableDocument>> : NSObject
+
+// Non-serialized document (or null)
```

Comment:
use this format for comments:
/**
 * Description here.
 *
 */

Reply:
Got it, comments updated.